test/system/250-systemd.bats

```diff
@@ -296,8 +296,6 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
 }
 
 @test "podman-kube@.service template" {
-    skip_if_remote "systemd units do not work with remote clients"
-
```

Comment:
Why is this removed? There is no point in testing this with podman-remote since the units have to use local podman.

Reply:
Global skip (in `setup()`). *EVERYTHING* in this file is skipped on remote. Having an explicit `skip` here serves only to confuse a reader.